functions/index.js

```diff
@@ -18,28 +19,82 @@ admin.initializeApp();
 
 
 
-const sections = ["AR", "CGC", "GC", "GM", "EL", "IN", "SV", "MA", "MT", "PH", "MX", "SIE", "SC"]
+const sections = ["AR", "CGC", "GC", "GM", "EL", "IN", "SV", "MA", "MT", "PH", "MX", "SIE", "SC", "NONE"]
 
+function set_error_callback (error) {
```

Comment:
Are theses messages ever shown to the end user?
If yes, it might be good to be more specific (like to contact the devs or so)